test_scripts/API/SetAudioStreamingIndicator/037_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_media_app_RPC_disallowed_by_user.lua

```diff
@@ -0,0 +1,245 @@
+          :Do(function()
+            local RequestIdListOfPermissions = self.hmiConnection:SendRequest("SDL.GetListOfPermissions", { appID = self.applications[config.application1.registerAppInterfaceParams.appName] })
+
+            EXPECT_HMIRESPONSE(RequestIdListOfPermissions)
+            :Do(function(_,data1)
+              local groups = {}
+              if #data1.result.allowedFunctions > 0 then
+                for i = 1, #data1.result.allowedFunctions do
+                  groups[i] = {
+                    name = data1.result.allowedFunctions[i].name,
+                    id = data1.result.allowedFunctions[i].id,
+                    allowed = true}
+                end
+              end
+              self.hmiConnection:SendNotification("SDL.OnAppPermissionConsent", { appID = self.applications[config.application1.registerAppInterfaceParams.appName], consentedFunctions = groups, source = "GUI"})
+              EXPECT_NOTIFICATION("OnPermissionsChange")
+            end)
+          end)
+        end
+      end)
+    end)
+  end)
+end
+
+function Test:Precondition_trigger_user_request_update_from_HMI()
+  testCasesForPolicyTable:trigger_user_request_update_from_HMI(self)
+end
+
+function Test:Precondition_SetAppTo_NONE()
+  self.hmiConnection:SendNotification("BasicCommunication.OnExitApplication", { appID = self.applications[config.application1.registerAppInterfaceParams.appName], reason = "USER_EXIT" })
+  EXPECT_NOTIFICATION("OnHMIStatus", { hmiLevel = "NONE" })
+end
+
+function Test:Precondition_PTU_revoke_app_group()
+  local RequestIdGetURLS = self.hmiConnection:SendRequest("SDL.GetURLS", { service = 7 })
+  EXPECT_HMIRESPONSE(RequestIdGetURLS,{result = {code = 0, method = "SDL.GetURLS", urls = {{url = "http://policies.telematics.ford.com/api/policies"}}}})
+  :Do(function()
+    self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest", { requestType = "PROPRIETARY", fileName = "filename"})
+
+    EXPECT_NOTIFICATION("OnSystemRequest", { requestType = "PROPRIETARY" })
+    :Do(function()
+      local CorIdSystemRequest = self.mobileSession:SendRPC("SystemRequest", { fileName = "PolicyTableUpdate", requestType = "PROPRIETARY"}, "SetAudioStreamingIndicator_group2.json")
+
+      EXPECT_HMICALL("BasicCommunication.SystemRequest")
+      :Do(function(_,data)
+        self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate", { policyfile = "/tmp/fs/mp/images/ivsu_cache/PolicyTableUpdate"})
+
+        local function to_run()
+          self.hmiConnection:SendResponse(data.id,"BasicCommunication.SystemRequest", "SUCCESS", {})
+        end
+        RUN_AFTER(to_run, 500)
+        self.mobileSession:ExpectResponse(CorIdSystemRequest, {success = true, resultCode = "SUCCESS"})
+      end)
+
+      EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATING"}, {status = "UP_TO_DATE"}):Times(2)
```

Comment:
@istoimenova, EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate") is better move higher, right after "BasicCommunication.OnSystemRequest" in order to have more stability